tools/integration_tests/run_tests_mounted_directory.sh

```diff
@@ -24,25 +24,7 @@ MOUNT_DIR=$2
 export CGO_ENABLED=0
 
 # Run integration tests for operations directory with static mounting
-gcsfuse --enable-storage-client-library=true --implicit-dirs=true $TEST_BUCKET_NAME $MOUNT_DIR
-GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/operations/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR
-sudo umount $MOUNT_DIR
-
-# Run test with persistent mounting
```

Comment:
dont we need this?

Reply:
The tests which are using enable_storage_client_library flag. I removed that.

Reply:
Got it. Need better commenting in this file to understand why tests are run and for which set of flags.

Reply:
Updated. PTAL.